exir/tests/test_memory_format_ops_pass_utils.py

```diff
@@ -99,10 +104,57 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
         return t1 * t2
 
 
+class AmbiguousDimOrderError(RuntimeError):
```

Comment:
I have a broader question is, why should be create this check?
This check went through every output tensor in the model and check whether its dim order is ambiguous. However, we should only care about the operator that involving dim order, otherwise
 a. ambiguous or not may not impact the behavior and
 b. may introduce extra false neagtive (e.g. slice op may introduce lots of ambiguous dim order tensor)